rocAL/rocAL/source/augmentations/color_augmentations/node_color_twist.cpp

```diff
@@ -45,7 +45,7 @@ void ColorTwistBatchNode::create_node()
     _hue.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
     _sat.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
 
-    _node = vxExtrppNode_ColorTwistbatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), _hue.default_array(), _sat.default_array(), _batch_size);/*A temporary fix for time being*/
+    // _node = vxExtrppNode_ColorTwistbatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), _hue.default_array(), _sat.default_array(), _batch_size);/*A temporary fix for time being*/
```

Comment:
why is this commented out?

Reply:
What is the temporary fix?

Reply:
@rrawther all of it is commented. I think it is so that the branch will build? Only the tensor implemented augmentations seems to call the rpp kernels. @fiona-gladwin is that what you are trying to do here?

Reply:
We have commented RPP calls for batchPD augmentations so that we can build the existing tensor branch. If we add all augmentations in this PR the changes will be more and hard to review. 
Like discussed we will issue subsequent PR's to convert batchPD to Tensor for rest of the augmentations.